beacon-chain/blockchain/service.go

```diff
@@ -283,10 +283,6 @@ func (s *Service) saveGenesisData(ctx context.Context, genesisState *pb.BeaconSt
 		return errors.Wrap(err, "Could not start fork choice service: %v")
 	}
 
-	if err := s.beaconDB.SaveGenesisBlockRoot(ctx, bytesutil.ToBytes32(s.FinalizedCheckpt().Root)); err != nil {
```

Comment:
confirmed `bytesutil.ToBytes32(s.FinalizedCheckpt().Root)` and `genesisBlockRoot` is the same 😆